### ISSUE TITLE
The AP value is small and the loss does not converge

### ISSUE THREAD

**Issue by little-little-star** [soo89/ISD-SSD#7]
Thanks for the great work! I tried to reproduce your results, but I encountered some problems, when I train train_isd.py and train_ssd.py,the loss would be greater than 100, and the network would retrain.However, when I reduce the learning rate, the loss function does not converge, and the AP value during training is only 7%. What should I do in this case

**Comment by little-little-star** [comment]
Why did I train train_isd. py with an AP value of 7%, not 70.2% at all?
